example.raml

```diff
@@ -21,31 +21,34 @@ securitySchemes:
             hashalg: sha512
             cookie_name: ramses_auth_tkt
             http_only: true
-    - read_only_users:
-        description: ACL that allows authenticated users to read collection
+    - item_owner_acl:
+        description: ACL that allows everyone to read, authenticated to create and item owners to edit item
         type: x-ACL
         settings:
             collection: |
                 allow g:admin all
-                allow authenticated view
+                allow everyone view,options
+                allow authenticated create
             item: |
                 allow g:admin all
-                allow authenticated view
-    - read_only_useritem_everyone:
-        description: ACL that allows everyone to read item, authenticated users to read collection
+                allow everyone view,options
```

Comment:
I think we need them for now. When ACL filtering will be reworked, these lines can be removed.